js/demo-apps/packages/react-front-end/package.json

```diff
@@ -8,7 +8,7 @@
     "build": "react-scripts build",
     "test": "react-scripts test",
     "test:url": "if [ -z ${APPLICATION_FRONT_END+x} ]; then echo 'http://localhost:3000'; else echo $APPLICATION_FRONT_END; fi",
-    "test:e2e": "cypress run --browser chrome $TEST_OPTIONS --config baseUrl=$(yarn run --silent test:url)",
+    "test:e2e": "cypress run --record --key 77270197-498a-4a68-b9ff-e5d6a0f04617 --browser chrome $TEST_OPTIONS --config baseUrl=$(yarn run --silent test:url)",
```

Comment:
this should be a secret value, and you should rotate this value (go delete it in your Cypress console). It's public now.